README.md

```diff
@@ -30,6 +30,27 @@ You can install `napari-micromanager` via [pip]:
 
     pip install napari-micromanager
 
+### Getting micromanager adapters:
+
+The easiest way to get the micromanager adapters is to use:
+
+```
+python -m pymmcore_plus.install
+```
+
+this will install micromanager to the pymmcore_plus folder in your site-package; use this to see where:
+
+```
+python -c "from pymmcore_plus import _util; print(_util.find_micromanager())"
```

Comment:
We should maybe expose this publicly on pymmcore-plus?

Reply:
agreed!